Dockerfile

```diff
@@ -64,3 +65,9 @@ WORKDIR /
 COPY --from=builder /workspace/backup-agent .
 USER nonroot:nonroot
 ENTRYPOINT ["/backup-agent"]
+
+FROM gcr.io/distroless/static:nonroot as restore-agent
+WORKDIR /
+COPY --from=builder /workspace/restore-agent .
+USER nonroot:nonroot
+ENTRYPOINT ["/restore-agent"]
```

Comment:
I moved the restore-agent build instructions here alongside the other docker images.